pkg/kubelet/prober/prober.go

```diff
@@ -200,13 +203,19 @@ func (pb *prober) runProbe(p *api.Probe, pod *api.Pod, status api.PodStatus, con
 		return pb.exec.Probe(pb.newExecInContainer(pod, container, containerID, p.Exec.Command))
 	}
 	if p.HTTPGet != nil {
+		scheme := strings.ToLower(string(p.HTTPGet.Scheme))
```

Comment:
I agree that calling ToLower is a bit weird, we should have a method or something, but we're not really mothod-y yet for API stuff.  OK for now.